### PR TITLE
Add ipycanvas recipe

### PR DESCRIPTION
**Pull request by martinRenou**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ipycanvas/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2019 Martin Renou
```
Comment (marcelotrevisani) [inline recipes/ipycanvas/LICENSE.txt]: The license is already being packaged. You can delete this file then.

Comment (martinRenou) [reply recipes/ipycanvas/LICENSE.txt]: Oh ok, thanks for the review!

**Comment by martinRenou** [comment]
Thanks !!

**Comment by martinRenou** [comment]
@marcelotrevisani There seems to have been an issue when creating the `ipycanvas-feedstock` repo... It is empty:
empty

**Pull request merged**